_includes/members/Timelab.html

```diff
@@ -0,0 +1,20 @@
+<div class="media">
+  <a href="{{ site.timelab_url }}">
+  <img class="media-object pull-left" src="{{page.root}}/img/members/timelab.png" alt="TimeLab Logo" />
+  </a>
+  <div id="timelab" class="media-body">
+    <a href="{{ site.timelab_url }}" class="project" target="_blank"> TimeLab </a>: we are a team of astrophysicists 
+    and software developers working together to build a variability analysis software to study black holes and fundamental 
+    physics under extreme conditions.
+    <br/>
+ <!-- repo -->
+                <a href="https://github.com/StingraySoftware/" target="_blank">
+              <span class="icon  icon--github">
+                <svg viewBox="0 0 16 16">
+                  <path fill="#333333" d="{{ site.gh_logo }}"/>
+                </svg>
+              </span>
+	      Timelab</a>
```

Comment:
Maybe you want also to add the slack or any mailing list?

Reply:
Done